### PR TITLE
Implemented HBase Pojo put operator and it's test case

### PR DESCRIPTION
**Pull request by brightchen**
commit f7176d147be1d7a6dc682783546e146eb522a94e
Author: brightchen <bright@datatorrent.com>
Date:   Tue Jun 9 15:20:54 2015 -0700

    MLHR-1728: Create concrete POJO-based output operator for HBase; MLHR-1730: HBase Operator should support batch

### CODE REVIEW

**Comment by brightchen** [comment]
implemented HBase Pojo put operator and it's test case.

File: contrib/src/main/java/com/datatorrent/contrib/common/FieldInfo.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  //Expressions set by user to get field values from input tuple.
+  @NotNull
+  private String columnExpression;
+
+  private SupportType type;
+  
+//  private static SupportType supportedTypes; // = new Class[]{ Boolean.class, Short.class, Integer.class, Long.class, Float.class, Double.class, String.class };
+//  private static final Map<String, Class> typeMap = new HashMap<String, Class>();
+//  static
+//  {
```
Comment (gauravgopi123) [inline contrib/src/main/java/com/datatorrent/contrib/common/FieldInfo.java]: Remove all this unwanted code.. 
Indentation is all messed up fix it

**Comment by brightchen** [comment]
modified code according to the comments

File: contrib/src/main/java/com/datatorrent/contrib/hbase/HBasePojoPutOperator.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
+package com.datatorrent.contrib.hbase;
```
Comment (PramodSSImmaneni) [inline contrib/src/main/java/com/datatorrent/contrib/hbase/HBasePojoPutOperator.java]: Add copyright

File: contrib/src/main/java/com/datatorrent/contrib/common/FieldInfo.java [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  /**
+   * the Java type of the column
+   */
+  public void setType(SupportType type)
+  {
+    this.type = type;
+  }
+
+  public static enum SupportType {
+    BOOLEAN(Boolean.class), SHORT(Short.class), INTEGER(Integer.class), LONG(Long.class), FLOAT(Float.class), DOUBLE(Double.class), STRING(String.class);
```
Comment (PramodSSImmaneni) [inline contrib/src/main/java/com/datatorrent/contrib/common/FieldInfo.java]: Can the java types be used directly instead of a wrapper

**Pull request merged**